src/VSCode/build.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+VERSION=${1:-"0.1.0"}
+
+# Navigate to extension directory
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)